modules/auth/auth_accounts.go

```diff
@@ -32,6 +33,42 @@ func GetGenesisAccounts(appState map[string]json.RawMessage, cdc codec.Marshaler
 	return accounts, nil
 }
 
+// GetGenesisVestingAccounts parses the given appState and returns the desired genesis vesting account details
+func GetGenesisVestingAccounts(appState map[string]json.RawMessage) ([]types.VestingAccount, error) {
+	log.Debug().Str("module", "auth (vesting)").Msg("parsing genesis")
+
+	var authState Accounts
+	if err := json.Unmarshal(appState[authttypes.ModuleName], &authState); err != nil {
+		return nil, err
+	}
+
+	// Store the accounts
+	accounts := []types.VestingAccount{}
+	for _, account := range authState.Accounts {
+		accountType := strings.Split(account.AccountType, ".")[3]
+
+		if accountType == "PeriodicVestingAccount" {
```

Comment:
We should store all types of vesting accounts, can you add them? 
-  ```BaseVestingAccount```
- ```ContinuousVestingAccount```
- ```DelayedVestingAccount```
- ```PeriodicVestingAccount```

Reply:
sure, I will add them :D